# CVE-2023-43642 CVE-2023-34454 CVE-2023-34453 CVE-2023-34455
# This is needed for transitive dep's in the two persistence-cassandra sub-modules
# This cannot be done via pombump because it adds scope and type fields
# that, for some reason, break the functionality and cause the snappy-java.jar
# files to either be missing in cql and or the other locations or an extra one to be added under persistence-api.
diff --git a/coordinator/pom.xml b/coordinator/pom.xml
index 6059b80f0..42adf974f 100644
--- a/coordinator/pom.xml
+++ b/coordinator/pom.xml
@@ -816,6 +816,11 @@
         <artifactId>nimbus-jose-jwt</artifactId>
         <version>9.21</version>
       </dependency>
+      <dependency>
+        <groupId>org.xerial.snappy</groupId>
+        <artifactId>snappy-java</artifactId>
+        <version>1.1.10.4</version>
+    </dependency>
     </dependencies>
   </dependencyManagement>
 </project>